script.js

```diff
@@ -303,3 +308,36 @@ function appendStory(story, board, storyUrl, statusBadge, estimate) {
     "</div>"
   );
 }
+
+function initializeDragAndDrop () {
+  $(".todo, .in-progress, .ready-for-review, .done").sortable({
+    connectWith: ".todo, .in-progress, .ready-for-review, .done"
+  });
+
+  $(".js-board").on("sortstop", function(event, ui) {
+      updatePivotal(ui);
+  });
+} //end initializeDragAndDrop
```

Comment:
You should be able to simplify this with (tested locally):

```
function initializeDragAndDrop () {
  $(".js-board").sortable({
    connectWith: ".js-board",
    stop: function(event, ui) {
      updatePivotal(ui);
    }
  });
} //end initializeDragAndDrop
```

Reply:
edited